### DISCUSSION TITLE
Showing error while using an audio file

### DISCUSSION THREAD

**Discussion by Rohan-IITG** [lagmoellertim/unsilence#105] [Q&A]
Hi,
The toolkit is working fine for video file. However, it is showing error for WAV file.

Error: FileNotFoundError: [Errno 2] No such file or directory: 'H:\\.tmp\\fd997a68-b042-417e-a5cd-952af46964fa\\out_final.wav

Kindly help me to solve the issue.

**Comment by lagmoellertim** [comment]
Did you try the `-ao` flag (audio only)? Maybe that helps. Otherwise, please send me a link to the file in order for me to retrace the problem

**Reply by Knackii** [reply]
Same issue and adding -ao fixed it! Maybe use the flag by default if a audio file is given as input ^^"
